config/webpack.config.js

```diff
@@ -10,13 +10,15 @@ require('babel-polyfill');
 
 const timestamp = new Date().getTime();
 
+const getAbsolutePath = (relativePath) => path.join(__dirname, '../', relativePath);
+
 const globalEntryFiles = {
-  styleConsolidated: './src/platform/site-wide/sass/style-consolidated.scss',
-  style: './src/platform/site-wide/sass/style.scss',
-  polyfills: './src/platform/polyfills/preESModulesPolyfills.js',
-  brandConsolidation: './src/platform/site-wide/sass/brand-consolidation.scss',
+  styleConsolidated: getAbsolutePath('src/platform/site-wide/sass/style-consolidated.scss'),
+  style: getAbsolutePath('src/platform/site-wide/sass/style.scss'),
+  polyfills: getAbsolutePath('src/platform/polyfills/preESModulesPolyfills.js'),
+  brandConsolidation: getAbsolutePath('src/platform/site-wide/sass/brand-consolidation.scss'),
   vendor: [
-    './src/platform/polyfills',
+    getAbsolutePath('src/platform/polyfills'),
```

Comment:
These changes are necessary so that Webpack can build the files even if it's not being executed from the `vets-website` project root.

Reply:
I'm not sure I understand this. I don't think we would separate the code from the build scripting required to build it, would we? Wouldn't the content build process grab assets that were already built if we were separating them?

Reply:
Before trying to explain, I want to preface this by saying that this has no effect on the current build. And we definitely need a spot for code that is safe for Metalsmith and Webpack to share.

But anyway, I hope I can answer your question and not just restate things you already knew. Let's definitely work together on this tomorrow if we can. I still have to create some sort of visual for this, but the mental model is something like this -

The content will grab assets that were already built during "CMS" deployments. But, during "full" deployments, it will need to execute the app-code's compilation as well, before following the rest of the CMS model as usual. In theory, we *could* have the app-code build totally separate from content build and then execute them individually with the content linking up after the app-code, but we also need a direct bridge between them for local development, and since they're developed directly for one another, it makes sense in my head to keep `script/build.js` as the central point of execution. However, even though they have that close relationship, the two separate deployment cycles - CMS and full - require a modular design.

There's also a some other concerns here -
- A CMS deployment is triggered any time a content-change is merged to master. What a "content-change" really means, and what effect intermediary code-merges could have, could be difficult to determine without a clear separation between files owned by the CMS versus the app-code.
- In version control, it would be valuable to preserve app-code versions in each commit from the content, so that we can easily checkout an earlier version of the site with the app-code exactly as it stood at that time. Kinda worried that information will only be visible on S3 logs, or by looking up an app-code commit by checking the dates.

No idea if I answered your question, but hope that helped in some way. This work is actually really close to being done. The next part is just incorporating the CMS deployment into the build, which I don't expect to be much.